cmd/cni-metrics-helper/metrics/prometheus.go

```diff
@@ -0,0 +1,179 @@
+package metrics
+
+import (
+	"net/http"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/prometheus/common/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
```

Comment:
Since these are the exact same metrics defined in https://github.com/aws/amazon-vpc-cni-k8s/blob/master/pkg/awsutils/awsutils.go#L86 and https://github.com/aws/amazon-vpc-cni-k8s/blob/master/pkg/ipamd/datastore/data_store.go#L89 , I think we can move the initialization of these variables to a common package. This will simplify adding new metrics in the future and keep everything in one place.